common-npm-packages/MSBuildHelpers/InvokeFunctions.ps1

```diff
@@ -108,7 +108,7 @@ function Invoke-MSBuild {
             # Hook up the custom logger.
             $loggerAssembly = "$PSScriptRoot\msbuildlogger\Microsoft.TeamFoundation.DistributedTask.MSBuild.Logger.dll"
             Assert-VstsPath -LiteralPath $loggerAssembly -PathType Leaf
-            $arguments = "$arguments /dl:CentralLogger,`"$loggerAssembly`";`"RootDetailId=$($detailId)|SolutionDir=$($solutionDirectory)`"*ForwardingLogger,`"$loggerAssembly`""
+            $arguments = "$arguments /dl:CentralLogger,`"$loggerAssembly`";`"RootDetailId=$($detailId)|SolutionDir=$($solutionDirectory)|enableOrphanedProjectsLogs=true`"*ForwardingLogger,`"$loggerAssembly`""
```

Comment:
Should the common package version be bumped as well, and for tasks which import this - also?

Reply:
There are two places which cause concern:
* the version has **_never_** been bumped up here (perhaps we can update it to `1.196.0`)
https://github.com/microsoft/azure-pipelines-tasks/blob/9a99462589ea2816fea28afbed111dd31297244f/common-npm-packages/package.json#L3
* the version has been bumped up here a couple of times (perhaps we can update it to `2.196.0`)
https://github.com/microsoft/azure-pipelines-tasks/blob/9a99462589ea2816fea28afbed111dd31297244f/common-npm-packages/MSBuildHelpers/package.json#L3

Reply:
We don't need to update the first - please bump the second. Using of sprint number make sense definitely - for better tracking.

Reply:
Done!